desktop/cmp/panel/PanelModel.ts

```diff
@@ -31,82 +102,59 @@ export class PanelModel extends HoistModel {
     //-----------------------
     // Immutable Properties
     //-----------------------
-    resizable;
-    resizeWhileDragging;
-    collapsible;
-    defaultSize;
-    minSize;
-    maxSize;
-    defaultCollapsed;
-    side;
-    renderMode;
-    refreshMode;
-    showSplitter;
-    showSplitterCollapseButton;
-    showHeaderCollapseButton;
-    showModalToggleButton;
-
-    @managed refreshContextModel;
-    @managed provider;
+    readonly resizable: boolean;
+    resizeWhileDragging: boolean;
```

Comment:
How come `resizeWhileDragging` isn't flagged as readonly here with the other immutable properties?

Reply:
we wrote a setter for it because it can be easily changed at runtime, I suppose.  Probably toolbox using that.  Moved the definition out of the immutable section for clarity